riscv-c-api.md

```diff
@@ -48,10 +48,41 @@ https://creativecommons.org/licenses/by/4.0/.
 | __riscv_fsqrt       | 1     | `F` extension is available and `-mno-fdiv` is not given.*[1]   |
 | __riscv_vector      | 1     | `V` extension is available.   |
 | __riscv_bitmanip    | 1     | `B` extension is available.   |
-| __riscv_zfh         | 1     | `Zfh` extension is available. |
 
 *[1] Not all compilers provide `-mno-div` and `-mno-fdiv` option.
 
+### Architecture Extension Test Marco
+
+Architecture extension test marco is a new set of test marco to checking the
+availability and version for certain extension, however not all compilers are
+supported, so you should check `__riscv_arch_test_marco` to make sure this
+compiler is supporting those preprocessor definitions.
+
+The value of architecture extension test marco are defined as its version,
+which is compute by following formula:
+
+```
+<MAJOR_VERSION> * 1,000,000 + <MINOR_VERSION> * 1,000 + <REVISION_VERSION>
+```
+
+For example:
+- F-extension v2.2 will define `__riscv_f` as `2002000`.
+- B-extension v0.92 will define `__riscv_b` as `92000`.
+
+| Name                    | Value        | When defined                  |
+| ----------------------- | ------------ | ----------------------------- |
+| __riscv_arch_test_marco | 1            | Defined if compiler support new architecture extension test marco. |
```

Comment:
It's better to move this comment before the table.  He it looks like a macro itself.

Reply:
Not sure you mean which comment? do you mind provide diff for that?